Tasks/FtpUploadV2/ftpuploadtask.ts

```diff
@@ -261,7 +261,13 @@ async function run() {
     tl.setResourcePath(path.join(__dirname, "task.json"));
 
     const tries = 3;
-    const ftpOptions: FtpOptions = getFtpOptions();
+    let ftpOptions: FtpOptions;
+    try {
+        ftpOptions = getFtpOptions();
+    } catch (err) {
+        tl.setResult(tl.TaskResult.Failed, tl.loc('InvalidFTPOptions', err.toString()));
```

Comment:
Nit: won't there be automatic type cast for error? Will it throw another exception if there will be err = null?

Reply:
Thanks, fixed!